### DISCUSSION TITLE
Design to Code

### DISCUSSION THREAD

**Discussion by ghost** [uxbox/uxbox#1613] [Show and tell]
Hi,

I saw your project recently and I'm excited about it.

Adding a code generation, i.e. component to code would be a cool feature and I have built code generator as part of my project  https://github.com/imvetri/ui-editor. Its a concept and can generate code in reactJS.

Code generation from Design is easier for Angular, react, svelete but not Vue. I guess it will be fun to have that feature.

Hit me up if you need any directions.

Congratulations on your project and thanks for sharing

-Vetrivel.

**Comment by nathfavour** [comment]
very good...I would be interested in working on this; as I'm also working on a fork of penpot that includes real time design to code preview, building, and export. see https://github.com/baIDEv-studio/baIDEv
